packages/portal/src/app/shared/errors/not-found/not-found.component.html

```diff
@@ -1 +1,11 @@
-<p>not-found works!</p>
+<div class="_404-container">
+    <h1>OOPS!</h1>
+    <h2 class="_404-header">Looks like we've strayed into uncharted territory</h2>
+    <div class="_404-child">
+        <button mat-raised-button routerLink="/about">Help</button>
+        <button mat-raised-button color="primary" routerLink="/home">Home</button>
+    </div>
+    <img src="assets/svgs/404-error.svg" alt="404 error image">
```

Comment:
Great work @grand-rick001 ! Being that this is an open source project under Azure Samples, we have some copyright compliance for images and other media assets that we need to observe. 

My suggestion is to remove the image file (although I love it!) since we can't add it to the repo at this time.

Reply:
Okay, thanks for the feedback, can I use another custom-made image/design instead? @anfibiacreativa

Reply:
Or maybe some copyright free images from [Pixabay](https://pixabay.com) and [Unsplash](https://unsplash.com)

Reply:
Understood, on it!🏃‍♂️